animations/lib/src/basics/08_hero_animation.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+class HeroAnimationDemo extends StatelessWidget {
+  static const String routeName = '/basics/hero_animation';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.lightBlue,
```

Comment:
A color suggestion:

* Rather than setting this background to lightBlue, leave it set to the default so it matches the other demos.
* Change the background of the Container holding the logo to a light gray, so it can be seen against the default scaffold color.
* Leave the background of HeroPage light blue, so it's more noticeable when it appears.

This will help the new demo blend into the others, and accentuate the appearance of HeroPage.